cli.js

```diff
@@ -10,6 +10,8 @@ import { fileURLToPath } from 'url';
 import { json1Presence } from './src/ot.js';
 import { randomId } from './src/randomId.js';
 import open from 'open';
+import ngrok from 'ngrok';
```

Comment:
I believe it should be possible for users to leverage NGrok without VZCode having any direct dependency on it.